torchtext/csrc/vectors.cpp

```diff
@@ -203,7 +181,7 @@ void parse_chunk(const std::string &file_path, size_t offset,
 
 std::tuple<IndexMap, StringList>
 _concat_vectors(std::vector<std::shared_ptr<StringList>> chunk_tokens,
-                int64_t num_header_lines, int64_t num_lines) {
+                const int64_t num_header_lines, const int64_t num_lines) {
```

Comment:
we are not passing by reference. Do we still need `const` here?

Reply:
Good call, I can remove this

Reply:
If it's `const`, it's better to be annotated as `const`. Compilers might find an opportunity to optimize something. Also that will prevent accidental overwrite.

Reply:
Readded